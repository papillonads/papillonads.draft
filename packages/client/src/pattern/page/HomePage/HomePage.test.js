import React from 'react'
import defaultRender from './HomePage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<HomePage />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
