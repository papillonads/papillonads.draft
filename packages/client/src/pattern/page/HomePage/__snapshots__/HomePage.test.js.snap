// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Snaphot must match defaultRender 1`] = `
<div
  className="homePage"
>
  <div />
</div>
`;
